com.unity.mobile.notifications/Runtime/Android/Plugins/mobilenotifications.androidlib/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'com.android.library'
+}
+
+android {
```

Comment:
buildToolsVersion '30.0.2' is specified in build.gradle of unityLibrary if built with Unity 2021.3 targeting SDK 33 (SDK is downloaded on request). buildToolsVersion is not specified for MobileNotifications and by default is requiring 29.0.2, but fails to install. Maybe we should match build tools version used for unityLibrary?

Reply:
It seems that build tools are not fixed, Unity uses what it finds in SDK.
I've installed latest 22.3 and build tools there are 32.0.0, looks to be the same in all later releases.
Worst part is, we don't expose it, so there is no way of knowing.